src/swarm/test.ts

```diff
@@ -0,0 +1,69 @@
+import { z } from 'zod';
```

Comment:
seems like this belongs in the `examples` folder since it's not library code?

Reply:
once it's ready for others, yes. could prob delete now but 🤷🏻‍♂️